warehouse/templates/accounts/two-factor.html

```diff
@@ -28,7 +31,7 @@ <h1 class="page-title">Two-factor authentication</h1>
 
     <div class="twofa-login">
       {% if has_webauthn %}
-      <div class="twofa-login__method {% if totp_form %}twofa-login__method--padded{% endif %}">
+      <div class="twofa-login__method {% if form %}twofa-login__method--padded{% endif %}">
```

Comment:
Is this change intentional? Looking at [the view code](https://github.com/pypa/warehouse/blob/master/warehouse/accounts/views.py#L207) seems `topt_form` was correct.

Reply:
thanks @yeraydiazdiaz - this was left over from some refactoring that it decided not to press ahead with.  I've removed it now.